src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Cohort/ChooseCohortViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Security.AccessControl;
+
+namespace SFA.DAS.ProviderCommitments.Web.Models.Cohort
+{
+    public class ChooseCohortViewModel
+    {
+        public long ProviderId { get; set; }
+
+        public string SortedByHeaderClassName { get; set; }
+
+        public const string HeaderClassName = "das-table__sort";
+
+        public ChooseCohortFilterModel FilterModel { get; set; } = new ChooseCohortFilterModel();
+
+        public void SortedByHeader()
```

Comment:
rather than a separate method to call, the logic within this method could reside in the property `get` of `SortedByHeaderClassName`

Reply:
changed